lib/trino-filesystem-cache-alluxio/src/main/java/io/trino/filesystem/alluxio/AlluxioInputHelper.java

```diff
@@ -48,6 +50,7 @@ public class AlluxioInputHelper
     private final String cacheKey;
     private final CacheManager cacheManager;
     private final AlluxioCacheStats statistics;
+    private final Optional<InputFileMetrics> metrics;
```

Comment:
This class should just expose a `long getCacheReadBytes()` method, no need for InputFileMetrics here